src/interface/sparse_pack.hpp

```diff
@@ -202,6 +201,27 @@ class SparsePack : public SparsePackBase {
       return SparsePack(SparsePackBase::GetPack(pmd, *this, include_block));
     }
 
+    template <class T>
+    SparsePack GetPack(T *pmd, bool only_fine_two_level_composite_blocks = true) const {
```

Comment:
Why is this additional function better than just passing the default argument to `include_block` above?

Reply:
IIRC, this structure saved creating a bunch of unnecessary intermediate vectors, which impacted MG performance to some extent.